src/main/java/com/centralesupelec/chowchow/search/controllers/TraktSearch.java

```diff
@@ -0,0 +1,55 @@
+package com.centralesupelec.chowchow.search.controllers;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+class Ids {
+    private int trakt;
+
+    public int getTrakt() {
+        return trakt;
+    }
+}
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+class TraktShow {
```

Comment:
This is definitively a DTO and it's used for the same purpose so I suggest we could rename it `TraktShowDTO` and use the same decorators and set the attributes to `final`.